src/components/RulerControls.vue

```diff
@@ -6,5 +6,5 @@ const activeToolStore = useRulerStore();
 </script>
 
 <template>
-  <LabelControls :labels-store="activeToolStore" />
+  <LabelControls :labelsStore="activeToolStore" />
```

Comment:
We need to establish a style convention, since we have both kebab case and camel case. I'm thinking about going with vue-strongly-recommended or vue-recommended eslint parsing, since I don't have a strong preference and this will enforce one of them. Thoughts?

Reply:
No strong option, but need to have the "attribute" camel case make typing work here.

Reply:
Interesting. I'll take a poke to see what's up with that.

Reply:
I updated the vue-tsc version, and it does not error with typed kebab case props now.

I'll remember to stick to kebab case component names.